src/main/java/org/dependencytrack/event/kafka/KafkaStreamsInitializer.java

```diff
@@ -85,6 +85,7 @@ static Properties getDefaultProperties() {
                 properties.put(SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG, Config.getInstance().getProperty(ConfigKey.KEY_STORE_PASSWORD));
             }
         }
+        properties.put(StreamsConfig.METRICS_RECORDING_LEVEL_CONFIG, Config.getInstance().getProperty(ConfigKey.KAFKA_STREAMS_METRICS_RECORDING_LEVEL));
```

Comment:
please could you also add this to the readme :) looks good

Reply:
Done!